application/core/plugins/statFunctions/countFunctions.php

```diff
@@ -80,4 +80,25 @@ public static function statCount($qCode, $submitted = true)
         }
         return intval(SurveyDynamic::model($surveyId)->count($oCriteria));
     }
+
+    /**
+     * Get the current valid survey id : Need to exist and activated
+     * Sometimes ExpressionManager don't get a valid survey id @see https://bugs.limesurvey.org/view.php?id=18191
+     * @return false[interger
+     */
+    private static function getCurrentSurveyId()
```

Comment:
This could be a core function?
`getExecutingSurveyId()`

Reply:
LimeSurvey API ? Good idea